R/plots.R

```diff
@@ -125,6 +125,133 @@ plot_pca <- function(obj,
   p
 }
 
+
+#' Plot Loadings and Interpretations 
```

Comment:
does this build without a proper description? there should be another line or more below with a more thorough description. make sure you are calling `document()` every time you introduce new documentation